templates/footer.html

```diff
@@ -0,0 +1,11 @@
+<footer class="page-footer">
+    <div class="footer-copyright">
+        <div class="container">
+            <span class="left">© 2017 {{ org_name }}</span>
+            <a class="grey-text text-lighten-4 right" href="https://github.com/{{ org_name }}"><i
```

Comment:
hmm, how does `"github.com{{ org_name }}"` end up as `github.com/coala` ?  django automatically adds the `/`? o_O